### ISSUE TITLE
Audio can be recorded while Jarvis is talking

### ISSUE THREAD

**Issue by cherche** [JaredCubilla/jarvis#26]
Audio can be recorded while Jarvis is talking, resulting in Jarvis's speech being recorded if the user has the audio input and output nearby. This can also result in multiple outputs from Jarvis being given at the same time.

**Comment by JaredCubilla** [comment]
This is true. I could change this to possibly stop all audio as soon as the user presses the record button. Thanks for finding this!
